src/components/__tests__/ConfirmationModal.spec.tsx

```diff
@@ -1,9 +1,10 @@
-import { fireEvent } from '@testing-library/react-native'
+import { fireEvent, RenderAPI } from '@testing-library/react-native'
 import React from 'react'
+import { Text } from 'react-native'
 
 import render from '../../testing/render'
 import ConfirmationModal, { ConfirmationModalProps } from '../ConfirmationModal'
-
+const childText = 'Children'
```

Comment:
(optional): Would be better to move this down (before renderLockingModal-function)